### DISCUSSION TITLE
Issue installing panel (blank page)

### DISCUSSION THREAD

**Discussion by Aeris1One** [pterodactyl/panel#2899] [Support] [answered]
Hi,
I just installed the panel following the docs (not the first time I install it). When going to my domain root it redirects well to /auth/login but the page appears fully white with nothing on it.

Panel logs : http://bin.ptdl.co/wug80
Nginx error logs (there's something in here) : http://bin.ptdl.co/u36m1

I tried at 14:49 (Paris time), so logs from 14:26 and earlier should not be taken into account.

**Comment by trenutoo** [comment, marked as answer]
The reason for the issue is provided in your logs:

SQLSTATE[HY000] [1045] Access denied for user 'pterodactyl'@'localhost' (using password: NO) (SQL: select * from `settings`)  

You have to resolve your database permission issues.

**Comment by Furanzou** [comment]
![60003D9B-D62F-4549-90DC-3A951E1F09F4](https://user-images.githubusercontent.com/83956844/147378921-91a1e284-6925-4402-a3e8-a2d7ec8a8871.png)
Same issue
